Makefile.am

```diff
@@ -84,9 +84,29 @@ EXTRA_DIST += debian/rules
 EXTRA_DIST += debian/source/format
 EXTRA_DIST += ucx.pc.in
 EXTRA_DIST += LICENSE
+
 endif #!DOCS_ONLY
 EXTRA_DIST += docs/uml/uct.dot
 
+.PHONY: lcov lcov-clean
+
+if HAVE_LCOV
+distclean-local: lcov-clean
+	find . -name "*.gcno" -type f -exec rm -f {} \;
+
+LCOV_DIR = lcov_data
+LCOV_INFO = lcov.info
+
+lcov-clean:
+	$(RM) -r $(LCOV_DIR)
+	find . -name "*.gcda" -type f -exec rm -f {} \;
```

Comment:
can we search only lcov dir?

Reply:
not really as gcda are apparently placed along with object files, but it is possible to build out/aside of tree also.